services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/ElasticQueryBuilder.groovy

```diff
@@ -0,0 +1,25 @@
+package org.openkilda.functionaltests.helpers
+
+import groovy.transform.builder.Builder
+import groovy.transform.builder.SimpleStrategy
+
+@Builder(builderStrategy = SimpleStrategy, prefix = "set")
+class ElasticQueryBuilder {
```

Comment:
Should be named ElasticQuery. The 'builder' will be generated for you by the `@Builder`